src/redux/categories/categories.js

```diff
@@ -0,0 +1,12 @@
+const CHECK_STATUS = './categories/CHECK_STATUS';
```

Comment:
- [ ] For name convention you can use duck pattern like this `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`